extensions/objects/templates/algebraic-expression-editor.component.ts

```diff
@@ -60,7 +60,7 @@ export class AlgebraicExpressionEditorComponent implements OnInit {
   }
 
   ngOnInit(): void {
-    if (this.value === undefined) {
+    if (this.value === undefined || this.value === null) {
```

Comment:
@iamprayush I have a doubt, do we also need this pattern here as well?https://github.com/oppia/oppia/blob/a171b393eed43b00d5029b62e5eeeb977bf977f8/extensions/objects/templates/ratio-expression-editor.component.ts#L46 & here
https://github.com/oppia/oppia/blob/a171b393eed43b00d5029b62e5eeeb977bf977f8/extensions/objects/templates/numeric-expression-editor.component.ts#L65

Reply:
Yup, I've updated them as well, thanks!